src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -248,6 +283,121 @@ def create(style_dataset, content_dataset, style_feature=None,
 
         _tkutl._print_neural_compute_device(cuda_gpus=cuda_gpus, use_mps=False,
                                             cuda_mem_req=cuda_mem_req, has_mps_impl=False)
+
+    if use_mps:
+        mps_key_map = {
```

Comment:
Have a large hard code dict in production (i.e. non-test code) may not be the best way of doing this. Conceptually what are the keys and values in dict?

Reply:
Thanks Toby, will refactor. Since I don't want to waste the use of the the Github runners. I'll close this PR and reopen when I'm ready to review.